Remove references to fabric (#3767)
alphagov/govuk-developer-docs: source/manual/alerts/rabbitmq-no-consumers-listening.html.md

### Troubleshooting steps

> 1. You could try restarting the application [on all the machines in a class](https://docs.publishing.service.gov.uk/manual/howto-run-ssh-commands-on-many-machines). After restarting, check to see if the problem is solved.

Reviewer: Suggest you signpost the `email_alert_api` machine class here.

```suggestion
1. You could try restarting the application [on all the `email_alert_api` machines](https://docs.publishing.service.gov.uk/manual/howto-run-ssh-commands-on-many-machines). After restarting, check to see if the problem is solved.
```
